File: ytrue-bean/src/main/java/com/ytrue/bean/dataobject/system/SysLog.java
package com.ytrue.bean.dataobject.system;

import com.baomidou.mybatisplus.annotation.TableName;
import com.ytrue.bean.dataobject.BaseIdEntity;
import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.ToString;

import java.io.Serial;
import java.time.LocalDateTime;

/**
 * @author ytrue
 * @description: SysLog
 * @date 2022/12/9 9:58
 */
@ToString(callSuper = true)
@EqualsAndHashCode(callSuper = true)
@Data
@TableName("sys_log")
public class SysLog extends BaseIdEntity {

    @Serial
    private static final long serialVersionUID = 7415492376407370163L;


    @Schema(description = "操作IP")
    private String requestIp;

    @Schema(description = "日志类型 OPT:操作类型;EX:异常类型")
    private String type;

    @Schema(description = "操作描述")
    private String description;

    @Schema(description = "操作人")
    private String operator;

    @Schema(description = "类路径")
    private String classPath;

    @Schema(description = "请求类型")
    private String actionMethod;

    @Schema(description = "请求地址")
    private String requestUri;

    @Schema(description = "请求类型 GET;POST;PUT;DELETE;PATCH;TRACE;HEAD;OPTIONS")
    private String httpMethod;

    @Schema(description = "请求参数")
    private String params;

    @Schema(description = "返回值")
    private String result;

    @Schema(description = "异常详情信息")
    private String exDesc;

    @Schema(description = "异常描述")
    private String exDetail;

    @Schema(description = "开始时间")
    private LocalDateTime startTime;

    @Schema(description = "结束时间")
    private LocalDateTime endTime;

    @Schema(description = "消耗时间")
    private Long consumingTime;

    @Schema(description = "浏览器")
    private String browser;


  
}

File: ytrue-admin-server-csharp/Ytrue.Models/Entities/System/SysLog.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Ytrue.Models.Entities.System;

/// <summary>
/// 操作日志实体
/// </summary>
[Table("sys_log")]
public class SysLog
{
    /// <summary>
    /// id
    /// </summary>
    [Key]
    public long Id { get; set; }

    /// <summary>
    /// 操作IP
    /// </summary>
    [Column("request_ip")]
    public string? RequestIp { get; set; }

    /// <summary>
    /// OPT:操作类型;EX:异常类型
    /// </summary>
    [Column("type")]
    public string? Type { get; set; }

    /// <summary>
    /// 操作人
    /// </summary>
    [Column("operator")]
    public string? Operator { get; set; }

    /// <summary>
    /// 操作描述
    /// </summary>
    [Column("description")]
    public string? Description { get; set; }

    /// <summary>
    /// 类路径
    /// </summary>
    [Column("class_path")]
    public string? ClassPath { get; set; }

    /// <summary>
    /// 请求方法
    /// </summary>
    [Column("action_method")]
    public string? ActionMethod { get; set; }

    /// <summary>
    /// 请求url
    /// </summary>
    [Column("request_uri")]
    public string? RequestUri { get; set; }

    /// <summary>
    /// 请求类型
    /// </summary>
    [Column("http_method")]
    public string? HttpMethod { get; set; }

    /// <summary>
    /// 请求参数
    /// </summary>
    [Column("params")]
    public string? Params { get; set; }

    /// <summary>
    /// 结果
    /// </summary>
    [Column("result")]
    public string? Result { get; set; }

    /// <summary>
    /// 异常详情信息
    /// </summary>
    [Column("ex_desc")]
    public string? ExDesc { get; set; }

    /// <summary>
    /// 异常描述
    /// </summary>
    [Column("ex_detail")]
    public string? ExDetail { get; set; }

    /// <summary>
    /// 浏览器
    /// </summary>
    [Column("browser")]
    public string? Browser { get; set; }

    /// <summary>
    /// 开始时间
    /// </summary>
    [Column("start_time")]
    public DateTime? StartTime { get; set; }

    /// <summary>
    /// 完成时间
    /// </summary>
    [Column("end_time")]
    public DateTime? EndTime { get; set; }

    /// <summary>
    /// 消耗时间
    /// </summary>
    [Column("consuming_time")]
    public long? ConsumingTime { get; set; }
}

